Add Channel Packing to PBR Guide (#108)
momentum-mod/docs: _posts/guides/2019-08-27-guide_the-pbr-shader.md

## Prerequisites
To make proper use of this Guide you should:
- Be on Momentum 0.8.4 or newer
- Be able to create PBR textures/work with the PBR Metal/Rough workflow
- Have either Substance Painter and/or Designer, and have some experience with them for making PBR textures from scratch

> - Know how to create textures such as Ambient Occlusion via baking if you're not using Substance to create them instead

Reviewer: ```suggestion
- Know how to create textures such as Ambient Occlusion via baking if you're not using Substance Painter and/or Designer to create them instead
```

> - Photoshop if you already have PBR textures (Metallic, Roughness, AO) that you just need to create an MRAO from

Reviewer: ```suggestion
- Photoshop if you're not using Substance Painter/Designer and you already have PBR textures (Metallic, Roughness, AO) that you need to create an MRAO from
```
I assume you don't need photoshop, rather it can be done in Substance painter/designer? If not, only remove the "just".

- Know how to create textures such as Ambient Occlusion via baking
- Be able to create PBR textures/work with the PBR Metal/Rough workflow
- Have either Substance Painter and/or Designer and have some experience with them  

> (Not all tools are covered but hopefully this guide may help with the general process)

Reviewer: Would change Substance Painter/Designer to be a recommendation rather than a requirement, since a little section is provided for doing this in Photoshop.

<img src="\assets\images\pbr_guide\pbr_designer_setup.png" alt="Painter Template" style="display: block; margin: auto;">

You should then have the 3 minimum textures needed for the shader.

## Manual MRAO Creation in Photoshop (Channel Packing)

> If you don't have something like Substance Designer to create an MRAO with, you can create it manually in something like Photoshop if you have several textures you want to create an MRAO from.

Reviewer: ```suggestion
If you have several textures you want to create an MRAO from and you don't have Substance Designer, you can create it manually in something like Photoshop.
```

The process is simple; you just need all 3 textures required to make an MRAO. The end result should look somewhat similar to this:

<img src="\assets\images\pbr_guide\channel_pack_final_result.jpg" alt="Final Resulting MRAO" style="display: block; margin: auto;">

The only layers you need are those of the 3 textures; add them all, check their properties, and look for these checkboxes:

<img src="\assets\images\pbr_guide\channel_pack_properties_dialog.jpg" alt="Final Resulting MRAO" style="display: block; margin: auto;">

By default, all 3 of these are checked, which is normal. Depending on which texture layer you've selected, change the checkboxes to the following:

Metallic:
- R Checked
- G Unchecked
- B Unchecked

Roughness:
- R Unchecked
- G Checked
- B Unchecked

Ambient Occlusion:
- R Unchecked
- G Unchecked
- B Checked

> Afterwards, you should have a strange, but colourful looking image that you can save and make use of in the PBR shader.

Reviewer: ```suggestion
Afterwards, you should have a strange, but colourful looking image that you can save and make use of with the PBR shader.
```
